python/ucxx/_lib/libucxx.pyx

```diff
@@ -527,9 +548,27 @@ cdef class UCXWorker():
 
         return info.decode("utf-8")
 
-    def get_address(self):
+    @property
+    def address(self):
         return UCXAddress.create_from_worker(self)
 
+    @property
+    def enable_delayed_submission(self):
+        return self._enable_delayed_submission
+
+    @property
+    def enable_python_future(self):
+        return self._enable_python_future
+
+    def get_address(self):
+        warnings.warn(
+            "UCXWorker.get_address() is deprecated and will soon be removed, "
+            "use the UCXWorker.address property instead",
+            DeprecationWarning,
```

Comment:
Just to note that if we want these to be visible to the user, they should be `FutureWarning` not `DeprecationWarning` (see https://docs.rapids.ai/api/cudf/stable/developer_guide/contributing_guide/#deprecating-and-removing-code and https://docs.python.org/3/library/warnings.html#warning-categories)

Reply:
Good point, fixed in https://github.com/rapidsai/ucxx/pull/158/commits/62d00b4829d96bea4542a0de8cc8934072c2ee6b